### PR TITLE
add ee-ais-query-vessel

### PR DESCRIPTION
**Pull request by chrieke**
Pull Request
============

### Scope of the PR

Add exactearth-ais-query-vessel, [block](https://marketplace.up42.com/block/75598899-26bf-4d1a-9a1c-b9d59bf0549a).

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — APPROVED** [review]
great!

File: source/up42-blocks/data/exactearth-ais-query-vessel.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+Output format
+-------------
+
+Output format is a ``GeoJSON`` file, see `GeoJSON <https://en.wikipedia.org/wiki/GeoJSON>`_ for a more detailed description.
+
+.. code-block:: javascript
+
```
Comment (perusio) [inline source/up42-blocks/data/exactearth-ais-query-vessel.rst:56]: Can we indent this code in a nicer way, e.g., using `jq` or jsonlint.com? Is somewhat hard to read.

**Review by perusio — CHANGES_REQUESTED** [review]
See below. Minor issues. Thx.

File: source/up42-blocks/data/exactearth-ais-query-vessel.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+or exactEarth Identifier (EEID) ids. It provides ship position point geometries and associated metadata e.g. ship name, cargo, destination port, etc.
+The output is a `ais-points` GeoJSON file containing the AIS point geometries and properties, and a `data.json` with the UP42 query parameters.
+
+
+Supported parameters
+--------------------
+
+For more information, please read the section :ref:`Data source query filters  <filters>`.
+
+* ``mmsi``: Ids of Maritime Mobile Service Identity (MMSI), e.g. ``["83273282", "94821299"]``.
```
[the remark this answers is not in the document]
Comment (chrieke) [reply source/up42-blocks/data/exactearth-ais-query-vessel.rst:28]: added mmsi link, no public source for eeid unfortunately, only in login api docs.

File: source/up42-blocks/data/exactearth-ais-query-vessel.rst (hunk 1 above)
Comment (chrieke) [reply source/up42-blocks/data/exactearth-ais-query-vessel.rst:57]: formatted, also for other ais blocks.

**Review by perusio — APPROVED** [review]
LGTM

**Pull request merged**